_health-care/_js/_components/error-message.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+/**
+ * An error message to be included conditionally by question components in an error state.
+ *
+ * @constructor
+ */
+class ErrorMessage extends React.Component {
+  render() {
+    return (
+      <span className="usa-input-error-message" id="input-error-message" role="alert">{this.props.message}</span>
```

Comment:
Right. Oops. We need the id if we want to include the `aria-describedby` on SSN (the values match to associate). Not sure how that'll work with the ids also needing to match, I guess we'll have to pass the id of the SSN as a prop to the error message? Is that a bad practice?